runtime/compiler/runtime/J9Runtime.hpp

```diff
@@ -158,7 +158,11 @@ typedef struct TR_AOTMethodHeader {
    uintptr_t compileMethodCodeSize;
    uintptr_t compileMethodDataStartPC;
    uintptr_t compileMethodDataSize;
+#if defined(JITSERVER_SUPPORT)
```

Comment:
We need to look whether we can eliminate this field. If not, since we buy the space anyway it makes more sense to leave the field without #ifdef and eliminate the `unused` field, adding a comment that this field is only used by JITServer.

Reply:
`compileFirstClassLocation` doesn't seem to be really used. Will remove `compileFirstClassLocation` and keep `unused` as the `master` branch.